packages/ubuntu_desktop_installer/lib/installer.dart

```diff
@@ -457,12 +456,16 @@ class _UbuntuDesktopInstallerWizard extends StatelessWidget {
 
   String? _nextStorageRoute(DiskStorageService service, dynamic arguments) {
     if (arguments == InstallationType.manual) {
-      return Routes.allocateDiskSpace;
+      return service.hasBitLocker
```

Comment:
this is covered by the integration test but we're not collecting test coverage because it essentially runs through the whole app so it'd be too easy to achieve ~100%